src/calculateBullsCows.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-console */
+'use strict';
+
+function calculateBullsCows(attempt, rightNumber) {
+  const result = {
+    bulls: 0,
+    cows: 0,
+  };
+
+  if (attempt.length !== 4) {
+    console.log('Enter a 4-digit number!');
+  }
+
+  if (attempt === rightNumber) {
+    console.log('Congratulations, you won!');
+
+    return -1;
```

Comment:
Or `-1` or an `object`, for me it could be a separate function that isn't related to the `calculateBullsCows` or just check.
Because it's strange if the function `calculateBullsCows` doesn't return any bulls/cows 🤷